x11-terms/ptyxis/ptyxis-46.3.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils meson xdg
+
+DESCRIPTION="Ptyxis is a terminal for GNOME with first-class support for containers."
```

Comment:
Maybe a more direct description as the one provided in the HOMEPAGE would be a bit better:
"A terminal for a container-oriented desktop"

Reply:
Done in https://github.com/gentoo/gentoo/pull/34844/commits/511e585bf14f96690a668da5a2fd3e26576884e2